### PR TITLE
Add brunch and update deps

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

File: app/public/js/cookieMessage.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+(function(global) {
+  'use strict'
+  var $ = global.jQuery;
+  var document = global.document;
+
+  function setCookie(name, value, options = {}) {
+    let cookieString = `${name}=${value}; path=/`;
```
Comment (markysoft) [inline app/public/js/cookieMessage.js]: I realise this code isn't new, but are `let` and string templates compatible with the minimum browser versions we support?

Comment (st3v3nhunt) [reply app/public/js/cookieMessage.js]: Its been cross browser tested so I would have thought so. However, the cookie banner is easy to miss so perhaps it doesn't work.

Comment (st3v3nhunt) [reply app/public/js/cookieMessage.js]: Confirmed this needs to work on IE 7+ so needs to be `var`.
Source https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#Browser_compatibility

**Comment by st3v3nhunt** [comment]
@markysoft Changes made.

**Comment by st3v3nhunt** [comment]
@markysoft Additional IE 7 backwards compatibility changes made.

**Pull request merged**